src/pages/index.tsx

```diff
@@ -1,17 +1,93 @@
 import React, { FunctionComponent } from "react";
 import Head from "next/head";
+import Aside from "../components/Aside";
+import { StartButton } from "../components/Button";
 
-const Home: FunctionComponent = () => {
-  return (
-    <>
-      <Head>
-        <title>Beacon Registration Service - Register a new beacon</title>
-      </Head>
-      <h1 className="govuk-heading-l">Header goes here</h1>
+const ServiceStartPage: FunctionComponent = () => (
+  <>
+    <Head>
+      <title>
+        Beacon Registration Service - Register a new 406 MHz distress beacon
+      </title>
+    </Head>
 
-      <p className="govuk-body">Service start page to go here!</p>
-    </>
-  );
-};
+    <Header />
```

Comment:
Very minor point, in the GOV.UK example the `<h1>` goes inside the `two-thirds` layout wrapper instead of before it, making the main content heading align with the aside heading:

![image](https://user-images.githubusercontent.com/54271433/106744724-08ee1100-6618-11eb-9c9d-189923ac5d85.png)

![image](https://user-images.githubusercontent.com/54271433/106744752-14413c80-6618-11eb-8c66-d4fc7bf07e9e.png)